src/room.py

```diff
@@ -1,2 +1,12 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+
+
+class Room():
+    def __init__(self, name, description, n_to="", s_to="", e_to="", w_to=""):
```

Comment:
Like we talked about in the 1:1 i think it would be better to use None here than empty strings for the default arguments